Detectors/MUON/MID/Workflow/src/RawDecoderSpec.cxx

```diff
@@ -53,7 +54,14 @@ class RawDecoderDeviceDPL
     auto tStart = std::chrono::high_resolution_clock::now();
 
     auto tAlgoStart = std::chrono::high_resolution_clock::now();
-    of::DPLRawParser parser(pc.inputs());
+
+    if (isDroppedTF(pc, header::gDataOriginMID)) {
+      return;
```

Comment:
@dstocco the whole point of this protection is that the device should send valid output in case the input allowing to start processing is not received. By doing nothing in case of lost TF you will block the downstream devices, which expect your input. Instead, your device should emit an "empty" output, with definition of "empty" suitable for your data types (I guess empty ROFRecord and channels vectors are OK).

Reply:
Hi @shahor02 , ok, I see. I send valid (empty) messages it in the new commit. It can be squashed before merging.